doc/source/Tutorials/Scripting/AddingAMenuItem/screengrab.py

```diff
@@ -1,21 +1,23 @@
-# BuildTarget: images/herd.png
+# BuildTarget: images/viewerCows.png
 
 import imath
 
 import GafferScene
-
 import GafferUI
 
 scriptWindow = GafferUI.ScriptWindow.acquire( script )
-
-script["Cow"] = GafferScene.SceneReader( "Cow" )
-script["Cow"]["fileName"].setValue( "${GAFFER_ROOT}/resources/cow/cow.scc" )
-script["Herd"] = GafferScene.Duplicate( "Herd" )
-script["Herd"]["target"].setValue( '/cow' )
-script["Herd"]["copies"].setValue( 7 )
-script["Herd"]["transform"]["translate"].setValue( imath.V3f( 16, 0, 0 ) )
-script["Herd"]["transform"]["rotate"].setValue( imath.V3f( 0, 45, 0 ) )
-script["Herd"]["in"].setInput( script["Cow"]["out"] )
-script.selection().add( script["Herd"] )
 viewer = scriptWindow.getLayout().editors( GafferUI.Viewer )[0]
-GafferUI.WidgetAlgo.grab( widget = viewer, imagePath = "images/herd.png" )
+
+# Circle of cows in Viewer
+sceneReaderNode = GafferScene.SceneReader( "Cow" )
```

Comment:
I started using variables to deal with some of the trickier bits of using the dictionary syntax:
- While it adds an extra line, it shortens the lines where the node is referenced. This helps with occasions where I'm several methods deep into several nested plugs – as I kept adding variables to help in those situations, I decided to just using them as standard.
- It removes the need to repeatedly remember/type out some of the more obtuse node names.
- It reduces the issues regarding accidentally calling the wrong node name, and anticipates cases where I would want to give a node a particular name for the screenshot that may not translate very well to code.
- It provides cleaner and more readable code (in my humble opinion).
- It makes the code more consistent, as I've been assigning variables to the various editors and windows (and using them quite frequently).
- It encourages people (mostly me, but also whoever decides to dig into this) to rely more on built-in methods and classes, rather than trying to do nefarious dictionary trickery (you might disagree).

As I'm not a professional coder, these reasons may be weak. To me, it overall feels like better code. Does it make it unreadable for you?

Aside: I wasn't aware of the node lifetime issue until long after this PR, but not `del`ing the variables in this case shouldn't hurt, should it? I'm not reusing them, and Gaffer's still closing between each `screengrab.py`.